src/drivers/GDI/Fl_GDI_Graphics_Driver.cxx

```diff
@@ -48,10 +48,9 @@ int Fl_GDIplus_Graphics_Driver::antialias() {
   return active;
 }
 
-#endif
-
-
-#if USE_GDIPLUS
+void Fl_GDIplus_Graphics_Driver::small_circle(int x, int y, int d, Fl_Color c) {
+  Fl_Graphics_Driver::small_circle(x, y, d, c);
+}
 
```

Comment:
Didn't you mention that the GDI+ driver doesn't need the special handling? If that is so, do we really need this overload that calls the (default) implementation back? Can't we just remove this one?

Reply:
This should remain because of the hierarchy of classes below `Fl_Graphics_Driver`: `Fl_GDIplus_Graphics_Driver` derives from `Fl_GDI_Graphics_Driver` which derives from `Fl_Scalable_Graphics_Driver`. We have to undo in `Fl_GDIplus_Graphics_Driver` what is done in `Fl_Scalable_Graphics_Driver` to get the desired behavior at the GDI+ level.

Reply:
OK, I understand.